Twoliter.toml

```diff
@@ -1,5 +1,5 @@
 schema-version = 1
-release-version = "1.38.0"
+release-version = "1.39.0"
```

Comment:
nit: you are also bumping the AMI version in this commit.

Reply:
Right, that is needed for the migration. I can separate them in a standalone commit if that is what you imply.